modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -154,6 +154,19 @@ do {
 ----
 ****
 
+[role="secondary"]
+.Cordova
+****
+Get the keycloak client object and refresh, if needed and still possible, the access token:
+
+[source,javascript]
+----
+authService.service.extract().updateToken(30).then(() => {
```

Comment:
`authService.service.extract()` - where is service coming from?

Reply:
Where authService comes from is documented into the `initalize the SDK` section.
Going to add a link to the keycloak docs